lib/api/gitlab.js

```diff
@@ -64,6 +64,15 @@ async function initRepo(repoName, token, endpoint) {
   if (endpoint) {
     process.env.GITLAB_ENDPOINT = endpoint;
   }
+  try {
+    logger.debug(`Determining Gitlab API version`);
+    // projects/visible route deprecated in v4
+    await glGot(`projects/visible`);
```

Comment:
Let's change to `projects/owned`, unless there is something smaller that was perhaps *added* in v4 we could use instead?